requirements.txt

```diff
@@ -1,2 +1,6 @@
 Adafruit-Blinka
-Adafruit_CircuitPython_ESP32SPI
```

Comment:
Same as before - I'd like to not add a requirement for esp32spi with this library to allow for new transport `socket`s  in the future.

Reply:
Done